src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
I don't want to move to one attribute set - there are clear cases where it will break existing configurations. We could do that in 2.4, but I don't see a good justification for breaking it, since there are equally good ways to handle without that.

I can see a few options:

- The duplicate `<fo:page-sequence>`, with different attribute sets. Doesn't break anybody, fixes the problem for everybody.
- I was playing with tables earlier this week and noted the `new-attr-set-reflection` named template, used to apply a new attribute set. That template seems kind of ugly but it's used a lot already for tables, and maybe that's a normal way to conditionally add attribute sets. So, instead of my original "If back matter, reset format", we could use "If back matter, add this back.matter.notices attribute set"
- Now wondering if I was over-complicating your original attribute set suggestion. My idea was to have all the logic in the notices attribute set -- "When back matter use 1, otherwise i". But I think perhaps it would work to inherit "i" from front matter -- thus preserving any customization -- and just use "if back matter reset to 1" inside the notices set. Hmm. That seems like it would work and not cause compatibility issues, right? Is that what you were thinking, rather than my `<xsl:choose>`?